client/js/shout.js

```diff
@@ -205,6 +207,7 @@ $(function() {
 			"action",
 		].indexOf(type) !== -1) {
 			switch (type) {
+			case "invite": data.msg.formattedAction = "invited you to"; break;
```

Comment:
IRC protocol does not support showing when someone invites to the channel.

However there is http://ircv3.net/specs/extensions/invite-notify-3.2.html